### PR TITLE
change tcc gateway name to tcc-gateway

### PR DESCRIPTION
**Pull request by rshriram**

### CODE REVIEW

File: install/kubernetes/helm/istio/values-tcc-gateway.yaml [hunk 1]
```diff
@@ -44,11 +44,11 @@ gateways:
   istio-egressgateway:
     enabled: false
 
-  tcc-app-gateway:
+  tcc-gateway:
     enabled: false
     labels:
-      app: istio-appgateway
-      tcc: appgateway
+      app: uniqueName
```
Comment (saigs) [inline install/kubernetes/helm/istio/values-tcc-gateway.yaml]: Do we want this name to be replaced? If so, could move to global variables?

File: install/kubernetes/helm/istio/values-tcc-gateway.yaml [hunk 2]
```diff
@@ -44,11 +44,11 @@ gateways:
...
+      tcc: gateway
```
Comment (saigs) [inline install/kubernetes/helm/istio/values-tcc-gateway.yaml]: Don't we want a common _tcc_ label for these istio deployments?

Comment (rshriram) [reply install/kubernetes/helm/istio/values-tcc-gateway.yaml]: yes.. thats tcc: gateway..

File: install/kubernetes/helm/istio/values-tcc-gateway.yaml (hunk 1 above)
Comment (rshriram) [reply install/kubernetes/helm/istio/values-tcc-gateway.yaml]: we can't. we need this to be literally a label in the gateway.. and per this helm chart, the labels:.. is taken and copied into the k8s deployment's labels..

File: install/kubernetes/helm/istio/values-tcc-gateway.yaml (hunk 2 above)
Comment (saigs) [reply install/kubernetes/helm/istio/values-tcc-gateway.yaml]: I mean, why don't we change `istio-ingressgateway` label to the same `tcc: gateway`?

Comment (rshriram) [reply install/kubernetes/helm/istio/values-tcc-gateway.yaml]: we dont want to rely on istio-ingressgateway. Its a shared gateway.. this is for both shared and non shared scenarios. Istio ingress gateway will have other junk ports over time. We dont want to keep culling them.

**Pull request merged**